ldap/servers/slapd/slapi-plugin.h

```diff
@@ -6087,7 +6087,7 @@ void slapi_lock_mutex(Slapi_Mutex *mutex);
 int slapi_unlock_mutex(Slapi_Mutex *mutex);
 Slapi_CondVar *slapi_new_condvar(Slapi_Mutex *mutex);
 void slapi_destroy_condvar(Slapi_CondVar *cvar);
-int slapi_wait_condvar(Slapi_CondVar *cvar, struct timeval *timeout);
+int slapi_wait_condvar(Slapi_CondVar *cvar, struct timeval *timeout);  /* DEPRECATED */
```

Comment:
If it's deprecated please add ` __attribute__((deprecated))` in the header dec.

Reply:
Changes made @Firstyear please review.